<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowShowRobot</class>
 <widget class="QMainWindow" name="MainWindowShowRobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simox - VirtualRobot - Show Robot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButtonReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadHand">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Hand</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>1100</y>
         <width>201</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>Joints</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>1130</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint Value</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="lcdNumberJointValue">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>1120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="smallDecimalPoint">
         <bool>true</bool>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
        <property name="value" stdset="0">
         <double>0.000000000000000</double>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSliderPos">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>1160</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="labelMinPos">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>1180</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>-1.57</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelMaxPos">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>1180</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>1.57</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxJoint">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxRobotNodeSet">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select RobotNodeSet</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>1070</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select RobotNode</string>
        </property>
       </widget>
      </widget>

      <widget class="QGroupBox" name="groupBox_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>201</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>Move Hand</string>
       </property>

       <widget class="QComboBox" name="comboBoxMoveHand">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>

       <widget class="QSlider" name="moveHandSlider1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="moveHandSlider2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="moveHandSlider3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

      <widget class="QSlider" name="moveHandSlider4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="moveHandSlider5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

      <widget class="QSlider" name="moveHandSlider6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

      </widget>

      <widget class="QGroupBox" name="groupBox_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>220</y>
         <width>201</width>
         <height>400</height>
        </rect>
       </property>
       <property name="title">
        <string>Grasps</string>
       </property>

       <widget class="QComboBox" name="comboBoxGrasp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QSlider" name="graspSlider9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>181</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="pageStep">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>

       <widget class="QPushButton" name="pushButtonGraspReset">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>

      <widget class="QPushButton" name="pushButtonCloseGrasp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Close Grasp</string>
       </property>
      </widget>

      <widget class="QPushButton" name="pushButtonMoveObject">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Object</string>
       </property>
      </widget>

      <widget class="QPushButton" name="pushButtonGraspOpt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>GraspOptimization</string>
       </property>
      </widget>

      <widget class="QPushButton" name="pushButtonMetric">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Metric Test</string>
       </property>
      </widget>

      <widget class="QComboBox" name="comboBoxGraspOpt">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>181</width>
          <height>22</height>
         </rect>
        </property>
       </widget>

      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
